.github/workflows/docker_management.prune.yml

```diff
@@ -0,0 +1,32 @@
+name: Prune temporary docker images
+
+on:    
+  schedule:
+    - cron:  '15 4 * * 6'
```

Comment:
This job is scheduled at the same time as `docker_management.branch.yml` job? is this intentional ?
Or many be this one should run after `docker_management.branch.yml`  job, because that job will created temp images tags?

Reply:
Yea, intentional just to remember/document easily. Prune job only prunes old images so it wouldn't really matter if prune job run after ocker_management.branch.yml